compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -127,7 +127,7 @@ internal class ComposeContainer(
     private val coroutineExceptionHandler = DesktopCoroutineExceptionHandler()
     private val coroutineContext = MainUIDispatcher + coroutineExceptionHandler
 
-    private val mediator = ComposeSceneMediator(
+    internal val mediator = ComposeSceneMediator(
```

Comment:
It should remain private to keep usages under control, `ComposeContainer` is supposed to encapsulate all interaction with the mediator. Please follow the existing pattern